src/Catalyst.Core.Modules.Dfs/Controllers/V0/StatsController.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Catalyst.Abstractions.Dfs.CoreApi;
+using Lib.P2P.Transports;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Catalyst.Core.Modules.Dfs.Controllers.V0
+{
+    /// <summary>
+    ///  Statistics for bitswap.
+    /// </summary>
+    public class StatsBitswapDto
+    {
+        /// <summary>
+        ///   TODO: Unknown.
+        /// </summary>
+        public int ProvideBufLen;
+
+        /// <summary>
+        ///   The content IDs that are wanted.
+        /// </summary>
+        public IEnumerable<BitswapLinkDto> Wantlist;
+
+        /// <summary>
+        ///   The known peers.
+        /// </summary>
+        public IEnumerable<string> Peers;
+
+        /// <summary>
+        ///   The number of blocks sent by other peers.
+        /// </summary>
+        public ulong BlocksReceived;
+
+        /// <summary>
+        ///   The number of bytes sent by other peers.
+        /// </summary>
+        public ulong DataReceived;
+
+        /// <summary>
+        ///   The number of blocks sent to other peers.
+        /// </summary>
+        public ulong BlocksSent;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4612924)